management-core/src/main/java/com/ns/task/product/service/ProductCoreServiceImpl.java

```diff
@@ -0,0 +1,57 @@
+package com.ns.task.product.service;
+
+import com.ns.task.model.ProductDTO;
+import com.ns.task.product.entities.ProductEntity;
+import com.ns.task.product.repositories.ProductRepository;
+import com.ns.task.services.ProductService;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.modelmapper.ModelMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+public class ProductCoreServiceImpl implements ProductService {
+
+    private final ModelMapper mapper;
+    private final ProductRepository repository;
+    private static final Logger logger = LogManager.getLogger();
+
+
+    @Autowired
+    public ProductCoreServiceImpl(ModelMapper mapper, ProductRepository repository) {
+        this.mapper = mapper;
+        this.repository = repository;
+    }
+
+    @Override
+    public List<ProductDTO> getProducts() {
+        List<ProductEntity> products = repository.findAll();
```

Comment:
Why are you using the `findAll` method instead of your stored procedure?

Reply:
fixed